src/scss/index.scss

```diff
@@ -208,7 +208,7 @@ table.field-list {
       position: sticky;
       top: calc(var(--pst-header-height) + 20px);
       z-index: 1000;
-      height: calc(100vh - 4rem);
+      height: calc(100vh - var(--pst-header-height) - 20px);
```

Comment:
`top` is defined just above as `calc(var(--pst-header-height) + 20px);`, so using the exact same value here to subtract from the full screen to get the height